RegressionIT.testAliasFields is failing

**Build scan**:
https://gradle-enterprise.elastic.co/s/vjebbutdxi44g
**Repro line**:
```
./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.RegressionIT.testAliasFields" -Dtests.seed=4C8CA3AF936CD948 -Dtests.security.manager=true -Dtests.locale=ko-KR -Dtests.timezone=Australia/Broken_Hill -Druntime.java=11
```
**Reproduces locally?**:
Yes
**Applicable branches**:
main, 7.9, 7.x
**Failure excerpt**:
```
org.elasticsearch.xpack.ml.integration.RegressionIT > testAliasFields FAILED
    java.lang.AssertionError: 
    Expected: a numeric value within <10.0> of <468.0>
         but: <478.1716820377456> differed by <0.17168203774559743> more than delta <10.0>
        at __randomizedtesting.SeedInfo.seed([4C8CA3AF936CD948:7AF2C5FC13CBA4AE]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.xpack.ml.integration.RegressionIT.testAliasFields(RegressionIT.java:553)
```

--- a/x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java
+++ b/x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java
@@ -36,7 +36,6 @@ import java.util.Set;
 
 import static org.elasticsearch.test.hamcrest.OptionalMatchers.isPresent;
 import static org.hamcrest.Matchers.anyOf;
-import static org.hamcrest.Matchers.closeTo;
 import static org.hamcrest.Matchers.emptyString;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThan;

--- a/x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java
+++ b/x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java
@@ -43,6 +43,7 @@ import static org.hamcrest.Matchers.greaterThan;
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
 import static org.hamcrest.Matchers.not;
 
 public class RegressionIT extends MlNativeDataFrameAnalyticsIntegTestCase {
@@ -477,7 +478,6 @@ public class RegressionIT extends MlNativeDataFrameAnalyticsIntegTestCase {
             "Finished analysis");
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/60212")
     public void testAliasFields() throws Exception {
         // The goal of this test is to assert alias fields are included in the analytics job.
         // We have a simple dataset with two integer fields: field_1 and field_2.
@@ -544,18 +544,25 @@ public class RegressionIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         startAnalytics(jobId);
         waitUntilAnalyticsIsStopped(jobId);
 
+        double predictionErrorSum = 0.0;
+
         SearchResponse sourceData = client().prepareSearch(sourceIndex).setSize(totalDocCount).get();
         for (SearchHit hit : sourceData.getHits()) {
             Map<String, Object> destDoc = getDestDoc(config, hit);
             Map<String, Object> resultsObject = getMlResultsObjectFromDestDoc(destDoc);
 
-            int featureValue = (int) destDoc.get("field_1");
-            double predictionValue = (double) resultsObject.get(predictionField);
-            assertThat(predictionValue, closeTo(2 * featureValue, 10.0));
-
             assertThat(resultsObject.containsKey(predictionField), is(true));
             assertThat(resultsObject.containsKey("is_training"), is(true));
+
+            int featureValue = (int) destDoc.get("field_1");
+            double predictionValue = (double) resultsObject.get(predictionField);
+            predictionErrorSum += Math.abs(predictionValue - 2 * featureValue);
         }
+
+        // We assert on the mean prediction error in order to reduce the probability
+        // the test fails compared to asserting on the prediction of each individual doc.
+        double meanPredictionError = predictionErrorSum / sourceData.getHits().getHits().length;
+        assertThat(meanPredictionError, lessThanOrEqualTo(10.0));
 
         assertProgressComplete(jobId);
         assertThat(searchStoredProgress(jobId).getHits().getTotalHits().value, equalTo(1L));

